W-11862525-first-draft-for-beta-site-build-CP (#114)
mulesoft/docs-apikit: modules/ROOT/pages/implementing-apikit-4-for-soap.adoc

= Implementing a SOAP Interface
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> Create an API by using Studio and APIkit for SOAP to consume a remote WSDL API. Download this link:{attachmentsdir}/tshirt2.wsdl[WSDL tshirt2.wsdl] example to create a SOAP interface that accesses data resources and integrates the interface with backend processes that function as an API.

Reviewer: ```suggestion
Create an API by using Anypoint Studio and APIkit for SOAP to consume a remote WSDL API. Download this {attachmentsdir}/tshirt2.wsdl[WSDL tshirt2.wsdl example] to create a SOAP interface that accesses data resources and integrates the interface with backend processes that function as an API.
```

Reviewer: I'm confused by the first sentence. Are you using Studio and APIkit to create an API that consumes a remote WSDL API?

Author: Yes! You're right!
Please, let me know if you need more info about this topic.
Thanks!

> Create an API by using Anypoint Studio and APIkit for SOAP to consume a remote WSDL API. Download this link:{attachmentsdir}/tshirt2.wsdl[WSDL tshirt2.wsdl example] to create a SOAP interface that accesses data resources and integrates the interface with backend processes that function as an API.

Reviewer 2: I would change this paragraph to something simpler like this and get rid of the instructions to download the example API which is already mentioned in "Before You Begin Using APIkit for SOAP" section.
```suggestion
Implement a WSDL API by using Anypoint Studio and APIkit for SOAP to consume a remote SOAP service. 
```

Author: OK! Thanks!

> Stat by xref:apikit-4-soap-project-task.adoc[].

Reviewer: This doesn't make sense to me.

Reviewer: If you use a link in context, add custom text to fix the capitalization. LMK if this is unclear.

Author: Hi, @hannanelson!
I've added a custom text to the link. Thanks!
Please, can you elaborate on the topic that does not make sense to you? Thanks!

Reviewer: @Crispy-Salesforce - "Stat by xxx". Should Stat be Start?

Author: Great catch! Thanks!

= Implementing a SOAP Interface
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> In this workflow, you create an API using Studio and APIkit for SOAP. APIkit for SOAP supports consuming a remote WSDL. This workflow uses a local link:{attachmentsdir}/tshirt2.wsdl[WSDL tshirt2.wsdl] example you must download. The SOAP interface you create accesses data resources and integrates the interface with backend processes that function as an API.

Reviewer 3: ```suggestion
In this workflow, you create an API by using Studio and APIkit for SOAP. APIkit for SOAP supports consuming a remote WSDL. This workflow uses a local link:{attachmentsdir}/tshirt2.wsdl[WSDL tshirt2.wsdl] example you must download. The SOAP interface that you create accesses data resources and integrates the interface with backend processes that function as an API.
```
```suggestion
In this workflow, you create an API using Studio and APIkit for SOAP. APIkit for SOAP supports consuming a remote WSDL. This workflow uses a local link:{attachmentsdir}/tshirt2.wsdl[WSDL tshirt2.wsdl] example you must download. The SOAP interface you create accesses data resources and integrates the interface with backend processes that function as an API.
```
```suggestion
In this workflow, you create an API using Studio and APIkit for SOAP. APIkit for SOAP supports consuming a remote WSDL. This workflow uses a local link:{attachmentsdir}/tshirt2.wsdl[WSDL tshirt2.wsdl] example you must download. The SOAP interface you create accesses data resources and integrates the interface with backend processes that function as an API.
```

Author: OK! Thanks!
